napari/_vispy/visuals/volume.py

```diff
@@ -127,10 +127,11 @@
             vec3 iloc = loc - step;
             for (int i=0; i<10; i++) {
                 color = $get_data(iloc);
-                if (floatNotEqual(color.g, categorical_bg_value) ) {
+                color = applyColormap(color.g);
+                if (floatNotEqual(color.a, 0) ) {
                     // when the non-background value is reached
                     // calculate the color (apply lighting effects)
-                    color = applyColormap(color.g);
+                    // color = applyColormap(color.g);
```

Comment:
I don't think there is a reason to keep this line in.

Reply:
I would also modify the above note, as "// check if value is different from the background value" is no longer true.

Update to something like "// check if the output value is transparent before drawing.""